content/blog/2021/07/ApacheCon-Asia-2021/index.md

```diff
@@ -0,0 +1,29 @@
+
+By leveraging [Knative](https://knative.dev/), [Quarkus](https://quarkus.io/)' fast runtime and [Camel K](/camel-k/latest/); we can implement a cloud-native, serverless integration platform: such as auto-scaling, shrink-to-zero and event-based communication capabilities, as well as the powerful integration capabilities of Apache Camel. You can build and maintain an army of Camels to solve your cloud-native problems with [Christina Lin](https://twitter.com/Christina_wm)'s [Building and Maintaining an Army of Camels to tackle your cloud native problems](https://apachecon.com/acasia2021/sessions/1070.html), which shows you how to develop a cloud-native integration application using [Camel Quarkus](/camel-quarkus/latest/)and [Camel K](/camel-k/latest/) with concrete examples from IDE tools, testing frameworks, scaling instances, configuration, CI/CD processes and finally monitoring.
+
+For those who want to learn more about the implementation details of [Camel Quarkus](/camel-quarkus/latest/) in Chinese, you can follow [the introduction of Camel Quarkus](https://apachecon.com/acasia2021/zh/sessions/1072.html) by [Feng Zheng](https://twitter.com/amos_zhengfeng), who will introduce the basic concept of [Camel Quarkus](/camel-quarkus/latest/)  and how to improve application loading performance by using [Quarkus](https://quarkus.io/)' build time optimization.
+
+With the popularity of event-driven architectures in cloud-based web services, services are increasingly being integrated through event response, and [Nicola Ferraro](https://twitter.com/ni_ferraro) has recently implemented a very handy set of [Kamelets](/camel-k/latest/kamelets/kamelets.html) on top of [Camel K](/camel-k/latest/). With Kamelets, developers can leverage the 300+ components provided by Apache Camel to Use yaml to define their own cloud-based application integration scripts and deploy them to k8s with the tools provided by [Camel K](/camel-k/latest/). For further details on using Kamelets, follow [Nicola Ferraro](https://twitter.com/ni_ferraro)'s talk [From Camel to Kamelets: new connectors for event-driven applications](https://apachecon.com/acasia2021/sessions/1073.html).
+
+[Tadayoshi Sato](https://twitter.com/tadayosi) has integrated Camel Route into [Jupyter](https://jupyter.org/) so that we can show beginners the [Jupyter](https://jupyter.org/) interactive documentation of the The Camel Route example can be shown to beginners through the [Jupyter](https://jupyter.org/) interactive documentation and can be used for hands-on practice based on interactive web pages. For those who would like to see the implementation in detail, you can follow [Tadayoshi Sato](https://twitter.com/tadayosi)'s talk [Integration with Scripting - Easy Integration with Apache Camel and JBang](https://apachecon.com/acasia2021/sessions/1074.html).
```

Comment:
Thanks for the introduction. However, this time I don't talk about my ICamel jupyter kernel project. Rather I'll talk about the idea of using Camel with JBang https://www.jbang.dev/ to make integrations easier like scripting.

Reply:
@WillemJiang please take note of this.

@tadayosi can you suggest a different paragraph here?

Reply:
Sorry, I just saw the comments.  @tadayosi Please feel free to update those content.

Reply:
Thanks, I saw your update but it still refers to Jupyter (which is not relevant to my session). How about this text?
> [Tadayoshi Sato](https://twitter.com/tadayosi) proposes the idea of integration scripting by combining Apache Camel with [JBang](https://jbang.dev). He demonstrates how easily we can write an intergration with the combination. For those who would like to know more about the idea, you can follow [Tadayoshi Sato](https://twitter.com/tadayosi)'s talk [Integration as a Script - Integration made easy with Apache Camel and JBang](https://apachecon.com/acasia2021/sessions/1074.html).